### DISCUSSION TITLE
Manual/UIE-uss-selector-precedence

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#689] [Notes]
# Manual/UIE-uss-selector-precedence

https://docs.unity3d.com/Manual/UIE-uss-selector-precedence.html

**Comment by vertxxyz** [comment]
> **The type of style sheet:** Selectors from user-defined style sheets takes precedence over selectors from default Unity style sheets.

This also refers to a [Theme Style Sheet](https://docs.unity3d.com/Manual/UIE-tss.html), which doesn't seem to count as "user-defined".
As you can use import statements in USS files I recommend making a main USS that acts as a theme instead if it's a hassle to have your theme constantly overridden by selectors with low precedence.
